### PR TITLE
Pu apache

### PR DESCRIPTION
**Pull request by kayiwa**
hey @jrgriffiniii all the molecule tests pass. I had no idea molecule is now a first class citizen in the Ansible ecosystem (Sept of last year). In any event. all the molecule tests work and I need to test CircleCI. 

If you have time to "pair" we can walk through all these changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,10 +6,42 @@ Ansible Demo
...
+This will create a virtualenv based on Python 3.7.1 under $(pyenv root)/versions
+in a directory called `ansible-demo-3.7.1`
+
+Activate your environment using the following command
+
+`pyenv activate ansible-demo-3.7.1`
+
+Then install all the required packages for this repo by running:
+
+`pip -r requirements.txt`
```
Comment (jrgriffiniii) [inline README.md]: I needed to execute `pip install -r requirements.txt` here

**Comment by jrgriffiniii** [comment]
Are we ultimately going to be using `tox`?  The documentation seems to be fairly biased towards using  this https://molecule.readthedocs.io/en/latest/testing.html#tox

**Comment by jrgriffiniii** [comment]
Installing the Python requirements works for me locally, however executing `molecule test` from the root path within the repository produces the following error:
```
pu-apache % molecule test
ERROR: 'molecule/*/molecule.yml' glob failed.  Exiting.
```

Changing into the directory with the tests does permit me to run the tests locally (and they all pass):
```
pu-apache % cd roles/pulibrary.apache2
pu-apache % molecule test
[...]
tests/test_apache2.py::test_hosts_file[ansible://instance] PASSED        [ 33%]
tests/test_apache2.py::test_is_apache_installed[ansible://instance] PASSED [ 66%]
tests/test_apache2.py::test_apache_listening_http[ansible://instance] PASSED [100%]
[...]
```

**Comment by kayiwa** [comment]
> Are we ultimately going to be using `tox`? The documentation seems to be fairly biased towards using this https://molecule.readthedocs.io/en/latest/testing.html#tox

If you say for 2020 then sure. ;-) We need to walk first I feel.

**Comment by jrgriffiniii** [comment]
I've attempted to resolve the CircleCI issue on my fork (https://github.com/kayiwa/ansible-demo/compare/pu-apache...jrgriffiniii:pu-apache-circleci?expand=1), but have encountered the following:
```
Validating schema /Users/griffinj/src/jrgriffiniii/ansible-demo/molecule/default/molecule.yml
ERROR: Failed to validate.
{'provisioner': [{'config_options': [{'defaults': [{'filter_plugins': ['disallowed user provided config option'], 'library': ['disallowed user provided config option'], 'roles_path': ['disallowed user provided config option']}]}]}]}
```

This may be related to https://github.com/ansible/molecule/issues/982 (I'm overriding the default `roles_path` within `config_options` in order to run the tests for the roles from outside of the directory), but I'm not certain.

**Comment by jrgriffiniii** [comment]
I believe that I've disabled Shippable on https://app.shippable.com/github/kayiwa/ansible-demo/settings, but I do not have the ability to remove integration from this repository (I lack the privileges).

**Comment by kayiwa** [comment]
> I believe that I've disabled Shippable on https://app.shippable.com/github/kayiwa/ansible-demo/settings, but I do not have the ability to remove integration from this repository (I lack the privileges).

I've yanked shippable configs.

**Pull request merged**